spec.bs

```diff
@@ -543,6 +590,8 @@ dictionary AuctionAdConfig {
   record<USVString, record<USVString, double>> perBuyerPrioritySignals;
   Promise<record<USVString, USVString>> perBuyerCurrencies;
   sequence<AuctionAdConfig> componentAuctions = [];
+  Promise<undefined> additionalBids;
+  USVString auctionNonce;
   AbortSignal? signal;

```

Comment:
Just a follow-up that we should file that I noticed while we are here: this `signal` should not be nullable `?`.